qa/L0_long_running_stress/stress.py

```diff
@@ -0,0 +1,612 @@
+                                          model_name,
+                                          dtype,
+                                          SEQUENCE_LENGTH_MEAN,
+                                          SEQUENCE_LENGTH_STDEV,
+                                          sequence_name=name)
+                    last_choices[client_idx] = "valid-no-end"
+                elif choice < 0.15:
+                    sequence_valid_valid(client_metadata,
+                                         rng,
+                                         trial,
+                                         model_name,
+                                         dtype,
+                                         SEQUENCE_LENGTH_MEAN,
+                                         SEQUENCE_LENGTH_STDEV,
+                                         sequence_name=name)
+                    last_choices[client_idx] = "valid-valid"
+                else:
+                    sequence_valid(client_metadata,
+                                   rng,
+                                   trial,
+                                   model_name,
+                                   dtype,
+                                   SEQUENCE_LENGTH_MEAN,
+                                   SEQUENCE_LENGTH_STDEV,
+                                   sequence_name=name)
+                    last_choices[client_idx] = "valid"
+
+    except Exception as ex:
+        _thread_exceptions_mutex.acquire()
+        try:
+            _thread_exceptions.append(traceback.format_exc())
+        finally:
+            _thread_exceptions_mutex.release()
+
+    # We need to explicitly close each client so that streams get
+    # cleaned up and closed correctly, otherwise the application
+    # can hang when exiting.
+    for c, i in client_metadata_list:
+        print("thread {} closing client {}".format(name, i))
+        c.close()
+
+    print("Exiting thread {}".format(name))
+
+
+def check_status(model_name):
+    client = grpcclient.InferenceServerClient("localhost:8001",
+                                              verbose=FLAGS.verbose)
+    stats = client.get_inference_statistics(model_name)
+    print(stats)
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-v',
+                        '--verbose',
+                        action="store_true",
+                        required=False,
+                        default=False,
+                        help='Enable verbose output')
+    parser.add_argument('-r',
+                        '--random-seed',
+                        type=int,
+                        required=False,
+                        help='Random seed.')
+    parser.add_argument('-t',
+                        '--concurrency',
+                        type=int,
+                        required=False,
+                        default=8,
+                        help='Request concurrency. Default is 8.')
+    parser.add_argument(
```

Comment:
Change from using iterations to using time as a measure of how long to run the test.